src/main/java/com/metamx/collections/bitmap/RoaringBitmapFactory.java

```diff
@@ -58,9 +61,7 @@ public MutableBitmap makeEmptyMutableBitmap()
   @Override
   public ImmutableBitmap makeEmptyImmutableBitmap()
   {
-    return new WrappedImmutableRoaringBitmap(
-        EMPTY_IMMUTABLE_BITMAP
-    );
+    return WRAPPED_IMMUTABLE_ROARING_BITMAP;
```

Comment:
@drcrallen 

Calling org.roaringbitmap.buffer.ImmutableRoaringBitmap#getIntIterator definitively does not return a shared iterator. A new iterator is returned each time:

https://github.com/lemire/RoaringBitmap/blob/master/src/main/java/org/roaringbitmap/buffer/ImmutableRoaringBitmap.java#L484

It should be perfectly safe to have multiple iterators running over the same bitmap (as long as this bitmap is immutable, that is).